.eslintrc.js

```diff
@@ -46,7 +46,7 @@ module.exports = {
     "no-shadow": "error",
     "no-unused-vars": "error",
     "no-var": "error",
-    "no-warning-comments": ["warn", {"terms": ["todo", "fixme", /* TODO: add "needs_doc", */ "xxx"], "location": "anywhere"}],
+    "no-warning-comments": ["warn", {"terms": ["todo", "fixme", "needs_doc", "xxx", "tbd"], "location": "anywhere"}],
```

Comment:
The "tbd" won't actually catch any of the TBDs in package.json since they're just strings. But, like, whatever.

Reply:
Like, whatever!